.builds/freebsd.yml

```diff
@@ -25,10 +26,9 @@ tasks:
   - build: |
       mkdir build
       cd build
-      CXXFLAGS="-DLIBICONV_PLUG" cmake ../gerbera \
-      -DCMAKE_VERBOSE_MAKEFILE:BOOL=ON \
+      CXXFLAGS="-DLIBICONV_PLUG" cmake ../gerbera -G Ninja \
       -DWITH_INOTIFY=1 -DWITH_MAGIC=1 -DWITH_MYSQL=0 -DWITH_CURL=1 -DWITH_JS=1 \
       -DWITH_TAGLIB=1 -DWITH_AVCODEC=1 -DWITH_FFMPEGTHUMBNAILER=1 \
       -DWITH_EXIF=1 -DWITH_LASTFM=0 -DWITH_SYSTEMD=0 -DWITH_DEBUG=1 -DWITH_TESTS=1
```

Comment:
Can you use or create some cmake preset here as well. Forgot that.

Reply:
Actually I’m curious why this is not using actions. vmactions supports FreeBSD.

Reply:
Yeah this isn't used looks like...